test/slicer/component.adb

```diff
@@ -0,0 +1,55 @@
+
+with Componolit.Gneiss.Log;
+with Componolit.Gneiss.Log.Client;
+with Componolit.Gneiss.Slicer;
+
+package body Component with
+   SPARK_Mode
+is
+
+   package Slicer is new Componolit.Gneiss.Slicer (Positive);
+
+   Log      : Gns.Log.Client_Session;
+   Alphabet : String (1 .. 26);
+
+   procedure Construct (Cap : Gns.Types.Capability)
+   is
+      S : Slicer.Context := Slicer.Create (Alphabet'First, Alphabet'Last, 5);
+      R : Slicer.Slice;
+   begin
+      Gns.Log.Client.Initialize (Log, Cap, "log_slicer");
+      if not Gns.Log.Initialized (Log) then
+         Main.Vacate (Cap, Main.Failure);
+         return;
+      end if;
+      for I in Alphabet'Range loop
+         Alphabet (I) := Character'Val (I + 64);
+      end loop;
+      --  R := Slicer.Get_Range (S);
+      --  pragma Assert (R.First = Alphabet'First);
+      --  pragma Assert (R.Last = Alphabet'Last);
+      --  pragma Assert (Slicer.Get_Length (S) = 5);
+      --  R := Slicer.Get_Slice (S);
+      --  pragma Assert (R.Last - R.First + 1 <= Slicer.Get_Length (S));
+      --  pragma Assert (Alphabet (R.First .. R.Last)'Length <= Slicer.Get_Length (S));
```

Comment:
Please add a comment or issue reference which this is still in here or remove the commented out code (same below).

Reply:
I added a clarification in https://github.com/Componolit/gneiss/pull/122/commits/2a4f3f2c831b191ba8bba037520b820e83266c08.